Nos/Views/Discover/DiscoverGrid.swift

```diff
@@ -0,0 +1,135 @@
+import SwiftUI
+import Dependencies
+
+struct DiscoverGrid: View {
+    @EnvironmentObject private var router: Router
+    @FetchRequest(fetchRequest: Author.fetchRequest()) var authors: FetchedResults<Author>
+    @ObservedObject var searchController: SearchController
+    @Dependency(\.relayService) private var relayService
+
+    // TODO: What's a better way to do this?
+    @State private var subscriptions = [ObjectIdentifier: SubscriptionCancellable]()
+    
+    @State private var selectedCategory: FeaturedAuthorCategory = .all
+
+    @Namespace private var animation
+
+    /// Initializes a DiscoverGrid with the selected category and a search controller.
+    /// - Parameters:
+    ///   - featuredAuthorCategory: The initial category of featured authors to display until
+    ///   the user changes the selection. Defaults to `.all` to show all featured authors.
+    ///   - searchController: The search controller to use for searching.
+    init(featuredAuthorCategory: FeaturedAuthorCategory = .all, searchController: SearchController) {
+        self.selectedCategory = featuredAuthorCategory
+
+        let authorsRequest = Author.allAuthorsRequest()
+        authorsRequest.fetchLimit = 100
```

Comment:
Do we need a fetchLimit? I think we want to get all authors in the list even if it's more than 100.

Reply:
No, I was just trying things out. I've removed it.